src/interfaces/src/plugin/common.h

```diff
@@ -156,4 +156,6 @@
 
 #define ARRAY_SIZE(arr) (sizeof(arr) / sizeof(arr[0]))
 
+#define MAX_IF_NAME_LEN IFNAMSIZ // 16 bytes
```

Comment:
Maybe unnecessary? If the constant already exists it should be used instead of creating a new one.